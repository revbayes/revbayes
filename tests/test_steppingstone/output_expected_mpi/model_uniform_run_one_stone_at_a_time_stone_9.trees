Iteration	Posterior	Likelihood	Prior	psi
0	-36113.41	-36115.84	2.435569	((Cheirogaleus_major[&index=4]:0.105440,(Pan_paniscus[&index=17]:0.009492,(((Colobus_guereza[&index=6]:0.110213,Tarsius_syrichta[&index=21]:0.162053)[&index=39]:0.063181,(Loris_tardigradus[&index=12]:0.042510,Galago_senegalensis[&index=8]:0.014504)[&index=29]:0.195186)[&index=34]:0.074090,(Lepilemur_hubbardorum[&index=11]:0.198209,Callicebus_donacophilus[&index=2]:0.038711)[&index=26]:0.027750)[&index=37]:0.096489)[&index=38]:0.074504)[&index=36]:0.143244,(Saimiri_sciureus[&index=20]:0.139621,(Propithecus_coquereli[&index=19]:0.071221,(Daubentonia_madagascariensis[&index=7]:0.455625,(((Cebus_albifrons[&index=3]:0.037977,Hylobates_lar[&index=9]:0.099765)[&index=41]:0.018840,((Perodicticus_potto[&index=18]:0.172876,Aotus_trivirgatus[&index=1]:0.223252)[&index=28]:0.016484,Microcebus_murinus[&index=14]:0.040151)[&index=35]:0.051267)[&index=31]:0.005151,(((Varecia_variegata_variegata[&index=22]:0.165441,(Lemur_catta[&index=10]:0.041624,Otolemur_crassicaudatus[&index=16]:0.166296)[&index=40]:0.017934)[&index=43]:0.180617,(Chlorocebus_aethiops[&index=5]:0.032326,Nycticebus_coucang[&index=15]:0.076331)[&index=24]:0.002276)[&index=32]:0.022114,Macaca_mulatta[&index=13]:0.186370)[&index=27]:0.086987)[&index=42]:0.051670)[&index=25]:0.156810)[&index=30]:0.082728)[&index=33]:0.054220,Galeopterus_variegatus[&index=23]:0.307090)[&index=44];
10	-31851.11	-31853.4	2.286856	(Galeopterus_variegatus[&index=23]:0.134014,Daubentonia_madagascariensis[&index=7]:0.119382,((((((((Hylobates_lar[&index=9]:0.125698,Callicebus_donacophilus[&index=2]:0.066381)[&index=37]:0.169648,Nycticebus_coucang[&index=15]:0.069134)[&index=41]:0.019558,Tarsius_syrichta[&index=21]:0.074381)[&index=28]:0.013207,(Perodicticus_potto[&index=18]:0.312548,((Lemur_catta[&index=10]:0.182762,Varecia_variegata_variegata[&index=22]:0.197938)[&index=43]:0.150895,Aotus_trivirgatus[&index=1]:0.177248)[&index=42]:0.042261)[&index=27]:0.133628)[&index=34]:0.030965,((Loris_tardigradus[&index=12]:0.065357,Colobus_guereza[&index=6]:0.070836)[&index=29]:0.148296,Galago_senegalensis[&index=8]:0.018625)[&index=39]:0.045246)[&index=35]:0.090967,Macaca_mulatta[&index=13]:0.067462)[&index=32]:0.024791,Cheirogaleus_major[&index=4]:0.043276)[&index=38]:0.058671,(((Pan_paniscus[&index=17]:0.012802,Lepilemur_hubbardorum[&index=11]:0.099936)[&index=26]:0.114631,((Microcebus_murinus[&index=14]:0.052716,(Chlorocebus_aethiops[&index=5]:0.029014,Otolemur_crassicaudatus[&index=16]:0.361907)[&index=36]:0.132207)[&index=40]:0.015445,(Cebus_albifrons[&index=3]:0.009622,Saimiri_sciureus[&index=20]:0.065498)[&index=31]:0.007484)[&index=24]:0.003643)[&index=25]:0.097772,Propithecus_coquereli[&index=19]:0.066946)[&index=30]:0.511656)[&index=33]:0.138196)[&index=44];
10	-31851.11	-31853.4	2.286856	(Galeopterus_variegatus[&index=23]:0.134014,Daubentonia_madagascariensis[&index=7]:0.119382,((((((((Hylobates_lar[&index=9]:0.125698,Callicebus_donacophilus[&index=2]:0.066381)[&index=37]:0.169648,Nycticebus_coucang[&index=15]:0.069134)[&index=41]:0.019558,Tarsius_syrichta[&index=21]:0.074381)[&index=28]:0.013207,(Perodicticus_potto[&index=18]:0.312548,((Lemur_catta[&index=10]:0.182762,Varecia_variegata_variegata[&index=22]:0.197938)[&index=43]:0.150895,Aotus_trivirgatus[&index=1]:0.177248)[&index=42]:0.042261)[&index=27]:0.133628)[&index=34]:0.030965,((Loris_tardigradus[&index=12]:0.065357,Colobus_guereza[&index=6]:0.070836)[&index=29]:0.148296,Galago_senegalensis[&index=8]:0.018625)[&index=39]:0.045246)[&index=35]:0.090967,Macaca_mulatta[&index=13]:0.067462)[&index=32]:0.024791,Cheirogaleus_major[&index=4]:0.043276)[&index=38]:0.058671,(((Pan_paniscus[&index=17]:0.012802,Lepilemur_hubbardorum[&index=11]:0.099936)[&index=26]:0.114631,((Microcebus_murinus[&index=14]:0.052716,(Chlorocebus_aethiops[&index=5]:0.029014,Otolemur_crassicaudatus[&index=16]:0.361907)[&index=36]:0.132207)[&index=40]:0.015445,(Cebus_albifrons[&index=3]:0.009622,Saimiri_sciureus[&index=20]:0.065498)[&index=31]:0.007484)[&index=24]:0.003643)[&index=25]:0.097772,Propithecus_coquereli[&index=19]:0.066946)[&index=30]:0.511656)[&index=33]:0.138196)[&index=44];
10	-31851.11	-31853.4	2.286856	(Galeopterus_variegatus[&index=23]:0.134014,Daubentonia_madagascariensis[&index=7]:0.119382,((((((((Hylobates_lar[&index=9]:0.125698,Callicebus_donacophilus[&index=2]:0.066381)[&index=37]:0.169648,Nycticebus_coucang[&index=15]:0.069134)[&index=41]:0.019558,Tarsius_syrichta[&index=21]:0.074381)[&index=28]:0.013207,(Perodicticus_potto[&index=18]:0.312548,((Lemur_catta[&index=10]:0.182762,Varecia_variegata_variegata[&index=22]:0.197938)[&index=43]:0.150895,Aotus_trivirgatus[&index=1]:0.177248)[&index=42]:0.042261)[&index=27]:0.133628)[&index=34]:0.030965,((Loris_tardigradus[&index=12]:0.065357,Colobus_guereza[&index=6]:0.070836)[&index=29]:0.148296,Galago_senegalensis[&index=8]:0.018625)[&index=39]:0.045246)[&index=35]:0.090967,Macaca_mulatta[&index=13]:0.067462)[&index=32]:0.024791,Cheirogaleus_major[&index=4]:0.043276)[&index=38]:0.058671,(((Pan_paniscus[&index=17]:0.012802,Lepilemur_hubbardorum[&index=11]:0.099936)[&index=26]:0.114631,((Microcebus_murinus[&index=14]:0.052716,(Chlorocebus_aethiops[&index=5]:0.029014,Otolemur_crassicaudatus[&index=16]:0.361907)[&index=36]:0.132207)[&index=40]:0.015445,(Cebus_albifrons[&index=3]:0.009622,Saimiri_sciureus[&index=20]:0.065498)[&index=31]:0.007484)[&index=24]:0.003643)[&index=25]:0.097772,Propithecus_coquereli[&index=19]:0.066946)[&index=30]:0.511656)[&index=33]:0.138196)[&index=44];
10	-31851.11	-31853.4	2.286856	(Galeopterus_variegatus[&index=23]:0.134014,Daubentonia_madagascariensis[&index=7]:0.119382,((((((((Hylobates_lar[&index=9]:0.125698,Callicebus_donacophilus[&index=2]:0.066381)[&index=37]:0.169648,Nycticebus_coucang[&index=15]:0.069134)[&index=41]:0.019558,Tarsius_syrichta[&index=21]:0.074381)[&index=28]:0.013207,(Perodicticus_potto[&index=18]:0.312548,((Lemur_catta[&index=10]:0.182762,Varecia_variegata_variegata[&index=22]:0.197938)[&index=43]:0.150895,Aotus_trivirgatus[&index=1]:0.177248)[&index=42]:0.042261)[&index=27]:0.133628)[&index=34]:0.030965,((Loris_tardigradus[&index=12]:0.065357,Colobus_guereza[&index=6]:0.070836)[&index=29]:0.148296,Galago_senegalensis[&index=8]:0.018625)[&index=39]:0.045246)[&index=35]:0.090967,Macaca_mulatta[&index=13]:0.067462)[&index=32]:0.024791,Cheirogaleus_major[&index=4]:0.043276)[&index=38]:0.058671,(((Pan_paniscus[&index=17]:0.012802,Lepilemur_hubbardorum[&index=11]:0.099936)[&index=26]:0.114631,((Microcebus_murinus[&index=14]:0.052716,(Chlorocebus_aethiops[&index=5]:0.029014,Otolemur_crassicaudatus[&index=16]:0.361907)[&index=36]:0.132207)[&index=40]:0.015445,(Cebus_albifrons[&index=3]:0.009622,Saimiri_sciureus[&index=20]:0.065498)[&index=31]:0.007484)[&index=24]:0.003643)[&index=25]:0.097772,Propithecus_coquereli[&index=19]:0.066946)[&index=30]:0.511656)[&index=33]:0.138196)[&index=44];
10	-31851.11	-31853.4	2.286856	(Galeopterus_variegatus[&index=23]:0.134014,Daubentonia_madagascariensis[&index=7]:0.119382,((((((((Hylobates_lar[&index=9]:0.125698,Callicebus_donacophilus[&index=2]:0.066381)[&index=37]:0.169648,Nycticebus_coucang[&index=15]:0.069134)[&index=41]:0.019558,Tarsius_syrichta[&index=21]:0.074381)[&index=28]:0.013207,(Perodicticus_potto[&index=18]:0.312548,((Lemur_catta[&index=10]:0.182762,Varecia_variegata_variegata[&index=22]:0.197938)[&index=43]:0.150895,Aotus_trivirgatus[&index=1]:0.177248)[&index=42]:0.042261)[&index=27]:0.133628)[&index=34]:0.030965,((Loris_tardigradus[&index=12]:0.065357,Colobus_guereza[&index=6]:0.070836)[&index=29]:0.148296,Galago_senegalensis[&index=8]:0.018625)[&index=39]:0.045246)[&index=35]:0.090967,Macaca_mulatta[&index=13]:0.067462)[&index=32]:0.024791,Cheirogaleus_major[&index=4]:0.043276)[&index=38]:0.058671,(((Pan_paniscus[&index=17]:0.012802,Lepilemur_hubbardorum[&index=11]:0.099936)[&index=26]:0.114631,((Microcebus_murinus[&index=14]:0.052716,(Chlorocebus_aethiops[&index=5]:0.029014,Otolemur_crassicaudatus[&index=16]:0.361907)[&index=36]:0.132207)[&index=40]:0.015445,(Cebus_albifrons[&index=3]:0.009622,Saimiri_sciureus[&index=20]:0.065498)[&index=31]:0.007484)[&index=24]:0.003643)[&index=25]:0.097772,Propithecus_coquereli[&index=19]:0.066946)[&index=30]:0.511656)[&index=33]:0.138196)[&index=44];
10	-31851.11	-31853.4	2.286856	(Galeopterus_variegatus[&index=23]:0.134014,Daubentonia_madagascariensis[&index=7]:0.119382,((((((((Hylobates_lar[&index=9]:0.125698,Callicebus_donacophilus[&index=2]:0.066381)[&index=37]:0.169648,Nycticebus_coucang[&index=15]:0.069134)[&index=41]:0.019558,Tarsius_syrichta[&index=21]:0.074381)[&index=28]:0.013207,(Perodicticus_potto[&index=18]:0.312548,((Lemur_catta[&index=10]:0.182762,Varecia_variegata_variegata[&index=22]:0.197938)[&index=43]:0.150895,Aotus_trivirgatus[&index=1]:0.177248)[&index=42]:0.042261)[&index=27]:0.133628)[&index=34]:0.030965,((Loris_tardigradus[&index=12]:0.065357,Colobus_guereza[&index=6]:0.070836)[&index=29]:0.148296,Galago_senegalensis[&index=8]:0.018625)[&index=39]:0.045246)[&index=35]:0.090967,Macaca_mulatta[&index=13]:0.067462)[&index=32]:0.024791,Cheirogaleus_major[&index=4]:0.043276)[&index=38]:0.058671,(((Pan_paniscus[&index=17]:0.012802,Lepilemur_hubbardorum[&index=11]:0.099936)[&index=26]:0.114631,((Microcebus_murinus[&index=14]:0.052716,(Chlorocebus_aethiops[&index=5]:0.029014,Otolemur_crassicaudatus[&index=16]:0.361907)[&index=36]:0.132207)[&index=40]:0.015445,(Cebus_albifrons[&index=3]:0.009622,Saimiri_sciureus[&index=20]:0.065498)[&index=31]:0.007484)[&index=24]:0.003643)[&index=25]:0.097772,Propithecus_coquereli[&index=19]:0.066946)[&index=30]:0.511656)[&index=33]:0.138196)[&index=44];
10	-31851.11	-31853.4	2.286856	(Galeopterus_variegatus[&index=23]:0.134014,Daubentonia_madagascariensis[&index=7]:0.119382,((((((((Hylobates_lar[&index=9]:0.125698,Callicebus_donacophilus[&index=2]:0.066381)[&index=37]:0.169648,Nycticebus_coucang[&index=15]:0.069134)[&index=41]:0.019558,Tarsius_syrichta[&index=21]:0.074381)[&index=28]:0.013207,(Perodicticus_potto[&index=18]:0.312548,((Lemur_catta[&index=10]:0.182762,Varecia_variegata_variegata[&index=22]:0.197938)[&index=43]:0.150895,Aotus_trivirgatus[&index=1]:0.177248)[&index=42]:0.042261)[&index=27]:0.133628)[&index=34]:0.030965,((Loris_tardigradus[&index=12]:0.065357,Colobus_guereza[&index=6]:0.070836)[&index=29]:0.148296,Galago_senegalensis[&index=8]:0.018625)[&index=39]:0.045246)[&index=35]:0.090967,Macaca_mulatta[&index=13]:0.067462)[&index=32]:0.024791,Cheirogaleus_major[&index=4]:0.043276)[&index=38]:0.058671,(((Pan_paniscus[&index=17]:0.012802,Lepilemur_hubbardorum[&index=11]:0.099936)[&index=26]:0.114631,((Microcebus_murinus[&index=14]:0.052716,(Chlorocebus_aethiops[&index=5]:0.029014,Otolemur_crassicaudatus[&index=16]:0.361907)[&index=36]:0.132207)[&index=40]:0.015445,(Cebus_albifrons[&index=3]:0.009622,Saimiri_sciureus[&index=20]:0.065498)[&index=31]:0.007484)[&index=24]:0.003643)[&index=25]:0.097772,Propithecus_coquereli[&index=19]:0.066946)[&index=30]:0.511656)[&index=33]:0.138196)[&index=44];
10	-31851.11	-31853.4	2.286856	(Galeopterus_variegatus[&index=23]:0.134014,Daubentonia_madagascariensis[&index=7]:0.119382,((((((((Hylobates_lar[&index=9]:0.125698,Callicebus_donacophilus[&index=2]:0.066381)[&index=37]:0.169648,Nycticebus_coucang[&index=15]:0.069134)[&index=41]:0.019558,Tarsius_syrichta[&index=21]:0.074381)[&index=28]:0.013207,(Perodicticus_potto[&index=18]:0.312548,((Lemur_catta[&index=10]:0.182762,Varecia_variegata_variegata[&index=22]:0.197938)[&index=43]:0.150895,Aotus_trivirgatus[&index=1]:0.177248)[&index=42]:0.042261)[&index=27]:0.133628)[&index=34]:0.030965,((Loris_tardigradus[&index=12]:0.065357,Colobus_guereza[&index=6]:0.070836)[&index=29]:0.148296,Galago_senegalensis[&index=8]:0.018625)[&index=39]:0.045246)[&index=35]:0.090967,Macaca_mulatta[&index=13]:0.067462)[&index=32]:0.024791,Cheirogaleus_major[&index=4]:0.043276)[&index=38]:0.058671,(((Pan_paniscus[&index=17]:0.012802,Lepilemur_hubbardorum[&index=11]:0.099936)[&index=26]:0.114631,((Microcebus_murinus[&index=14]:0.052716,(Chlorocebus_aethiops[&index=5]:0.029014,Otolemur_crassicaudatus[&index=16]:0.361907)[&index=36]:0.132207)[&index=40]:0.015445,(Cebus_albifrons[&index=3]:0.009622,Saimiri_sciureus[&index=20]:0.065498)[&index=31]:0.007484)[&index=24]:0.003643)[&index=25]:0.097772,Propithecus_coquereli[&index=19]:0.066946)[&index=30]:0.511656)[&index=33]:0.138196)[&index=44];
10	-31851.11	-31853.4	2.286856	(Galeopterus_variegatus[&index=23]:0.134014,Daubentonia_madagascariensis[&index=7]:0.119382,((((((((Hylobates_lar[&index=9]:0.125698,Callicebus_donacophilus[&index=2]:0.066381)[&index=37]:0.169648,Nycticebus_coucang[&index=15]:0.069134)[&index=41]:0.019558,Tarsius_syrichta[&index=21]:0.074381)[&index=28]:0.013207,(Perodicticus_potto[&index=18]:0.312548,((Lemur_catta[&index=10]:0.182762,Varecia_variegata_variegata[&index=22]:0.197938)[&index=43]:0.150895,Aotus_trivirgatus[&index=1]:0.177248)[&index=42]:0.042261)[&index=27]:0.133628)[&index=34]:0.030965,((Loris_tardigradus[&index=12]:0.065357,Colobus_guereza[&index=6]:0.070836)[&index=29]:0.148296,Galago_senegalensis[&index=8]:0.018625)[&index=39]:0.045246)[&index=35]:0.090967,Macaca_mulatta[&index=13]:0.067462)[&index=32]:0.024791,Cheirogaleus_major[&index=4]:0.043276)[&index=38]:0.058671,(((Pan_paniscus[&index=17]:0.012802,Lepilemur_hubbardorum[&index=11]:0.099936)[&index=26]:0.114631,((Microcebus_murinus[&index=14]:0.052716,(Chlorocebus_aethiops[&index=5]:0.029014,Otolemur_crassicaudatus[&index=16]:0.361907)[&index=36]:0.132207)[&index=40]:0.015445,(Cebus_albifrons[&index=3]:0.009622,Saimiri_sciureus[&index=20]:0.065498)[&index=31]:0.007484)[&index=24]:0.003643)[&index=25]:0.097772,Propithecus_coquereli[&index=19]:0.066946)[&index=30]:0.511656)[&index=33]:0.138196)[&index=44];
10	-31851.11	-31853.4	2.286856	(Galeopterus_variegatus[&index=23]:0.134014,Daubentonia_madagascariensis[&index=7]:0.119382,((((((((Hylobates_lar[&index=9]:0.125698,Callicebus_donacophilus[&index=2]:0.066381)[&index=37]:0.169648,Nycticebus_coucang[&index=15]:0.069134)[&index=41]:0.019558,Tarsius_syrichta[&index=21]:0.074381)[&index=28]:0.013207,(Perodicticus_potto[&index=18]:0.312548,((Lemur_catta[&index=10]:0.182762,Varecia_variegata_variegata[&index=22]:0.197938)[&index=43]:0.150895,Aotus_trivirgatus[&index=1]:0.177248)[&index=42]:0.042261)[&index=27]:0.133628)[&index=34]:0.030965,((Loris_tardigradus[&index=12]:0.065357,Colobus_guereza[&index=6]:0.070836)[&index=29]:0.148296,Galago_senegalensis[&index=8]:0.018625)[&index=39]:0.045246)[&index=35]:0.090967,Macaca_mulatta[&index=13]:0.067462)[&index=32]:0.024791,Cheirogaleus_major[&index=4]:0.043276)[&index=38]:0.058671,(((Pan_paniscus[&index=17]:0.012802,Lepilemur_hubbardorum[&index=11]:0.099936)[&index=26]:0.114631,((Microcebus_murinus[&index=14]:0.052716,(Chlorocebus_aethiops[&index=5]:0.029014,Otolemur_crassicaudatus[&index=16]:0.361907)[&index=36]:0.132207)[&index=40]:0.015445,(Cebus_albifrons[&index=3]:0.009622,Saimiri_sciureus[&index=20]:0.065498)[&index=31]:0.007484)[&index=24]:0.003643)[&index=25]:0.097772,Propithecus_coquereli[&index=19]:0.066946)[&index=30]:0.511656)[&index=33]:0.138196)[&index=44];
