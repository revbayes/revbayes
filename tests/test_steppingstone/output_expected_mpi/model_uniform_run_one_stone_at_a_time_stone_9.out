state	power	likelihood
4	1.69351e-05	-31094.5
5	1.69351e-05	-29376.7
6	1.69351e-05	-28980.1
7	1.69351e-05	-28253.5
8	1.69351e-05	-29184.8
9	1.69351e-05	-29613
10	1.69351e-05	-31853.4
